initfiles/componentfiles/configschema/xsd/esp_service_smc.xsd

```diff
@@ -13,6 +13,19 @@
             <xs:attribute name="warnIfFreeMemoryUnder" type="xs:nonNegativeInteger" use="optional" default="5" hpcc:displayName="Warn Free Memory" hpcc:group="Monitoring" hpcc:tooltip="Available memory under this value is flagged as warning in monitoring output"/>
         </xs:attributeGroup>
 
+        <xs:attributeGroup name="elk">
```

Comment:
ELK Integration

Reply:
This is a reference name only for inclusion elsewhere in the schema. Not a human readable value.

Reply:
ok, then elkintegration